src/libmscore/instrtemplate.cpp

```diff
@@ -334,6 +352,9 @@ void InstrumentTemplate::write(XmlWriter& xml) const
             ma.write(xml);
         }
     }
+    if (!family) {
+        xml.tag("family", family->id);
+    }
```

Comment:
see https://github.com/musescore/MuseScore/pull/6479#pullrequestreview-582320169 

needs sync after fix in the 3.x

Reply:
Solved in PR #7405.